enterprise/internal/batches/service/service_test.go

```diff
@@ -51,23 +52,39 @@ func TestServicePermissionLevels(t *testing.T) {
 	admin := bt.CreateTestUser(t, db, true)
 	user := bt.CreateTestUser(t, db, false)
 	otherUser := bt.CreateTestUser(t, db, false)
+	otherUser2 := bt.CreateTestUser(t, db, false)
```

Comment:
Since we're using it primarily for this maybe:

```suggestion
	nonOrgMember := bt.CreateTestUser(t, db, false)
```